wiki/messaging.md

```
@@ -181,20 +181,20 @@ GoBricks applies production-safe AMQP reconnection defaults when messaging is co
-| `reconnect.max_delay` | 60s | Maximum backoff cap for exponential retry |
-| `publisher.max_cached` | 50 | Maximum cached publisher channels |
-| `publisher.idle_ttl` | 10m | TTL for idle publisher channels |
+| `reconnect.reinitdelay` | 2s | Delay between channel re-initialization |
+| `reconnect.resenddelay` | 5s | Delay before resending failed messages |
+| `reconnect.connectiontimeout` | 30s | Timeout for connection establishment |
+| `reconnect.maxdelay` | 60s | Maximum backoff cap for exponential retry |
+| `publisher.maxcached` | 50 | Maximum cached publisher channels |
+| `publisher.idlettl` | 10m | TTL for idle publisher channels |
 
 **Override defaults** in `config.yaml`:
 ```yaml
```

markdownlint-cli2 (0.22.1)

[warning] Fenced code blocks should be surrounded by blank lines

(MD031, blanks-around-fences)